layer/composables/breacrumbs.ts

```diff
@@ -26,12 +40,26 @@ function getBreadcrumbs(input: string) {
   return stepNode(startNode)
 }
 
-export function useBreadcrumbs() {
+interface UseBreadcrumbsOptions {
+  useI18n?: boolean
+  keepLangPrefix?: boolean
+  translationPrefix?: string
+}
+
+export function useBreadcrumbs(options: UseBreadcrumbsOptions) {
   const router = useRouter()
+  const opts = unref(options) || { translationPrefix: 'pages' }
+  let getBreadcrumbsOptions: GetBreadcrumbsOptions
+  let $t: ((args: string) => void) | undefined
+  if (opts.useI18n) {
+    const { t, localeProperties } = useI18n()
+    $t = t
+    getBreadcrumbsOptions = { localeProperties, keepLangPrefix: opts.keepLangPrefix }
+  }
   return computed(() => {
     const routes = router.getRoutes()
     const route = router.currentRoute.value
-    return getBreadcrumbs(route.path)
+    return getBreadcrumbs(route.path, getBreadcrumbsOptions)
       .reverse()
       .map(path => ({
         path,
```

Comment:
Hi, the pull-request helped me, thank you so much.
By the way, regarding the acquisition of meta in the next line, it seems that meta cannot be referenced because it cannot match dynamic routes such as /foo/bar/:params, so instead ```use router.resolve(path)?.meta``` maybe better.